### PR TITLE
Add `Style/EmptyElse` in `.rubocop.yml`

### PR DESCRIPTION
**Pull request by kamipo**
To avoid #26375 and #27170 in the future.

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -24,6 +24,10 @@ Style/CaseIndentation:
 Style/CommentIndentation:
   Enabled: true
 
+# No empty else.
```
Comment (utilum) [inline .rubocop.yml]: 👍

**Comment by kaspth** [comment]
I'm not sure we should codify this because one stray (truly) empty else was left over.

They can also be used to improve the readability as @fxn shows here: https://github.com/kamipo/rails/commit/e86524c0c5a26ceec92895c830d1355ae47a7034

I'll close for now then @fxn can override if he feels the need to. Thanks! 😊

**Comment by fxn** [comment]
@kaspth 👍

**Pull request closed without merging**